### DISCUSSION TITLE
A few balance requests

### DISCUSSION THREAD

**Discussion by warringgod** [sleepeybunney/maelstrom#33] [Ideas]
Is there a way to make it so you only steal/drop 1 of an item? getting 9 of a super powerful item can break the game a bit.

Make stat boosts very expensive, say 100000? (currently sometimes 10) Holy war and other invincibility items should also be extremely expensive or not in the shops at all in my opinion.
A few other items like flare, ultima stones also sell for 10

**Comment by sleepeybunney** [comment]
Well the good news is you can't get 9 items any more... I changed the maximum to 8 ;p

But yeah you'll be able to set your own cap on it like with the drawn spells from monsters, probably next update. A review of all the shop prices is on the todo list as well.
